tests/main/mount-order-regression/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Regression test for snap-update-ns mount order
```

Comment:
should that go in the regression suite instead of main?

Reply:
Ah, I didn't notice we have a `regression` suite. Indeed! I now moved it there, thanks!

Reply:
awesome, yaw